### DISCUSSION TITLE
How to use the plugin "measured_polarized" to call my own pBRDF dataset

### DISCUSSION THREAD

**Discussion by helloKOGOU** [mitsuba-renderer/mitsuba3#695] [Q&A] [answered]
I want to do polarization rendering and the lab is equipped to measure pBRDF data, but I don't know how to use this plugin. I also looked at the database provided in the official documentation, but I don't know how that file is wrapped into a .pbrdf file.
I would be very grateful if you could help me.
<img width="433" alt="a1f6b87817fd26b922b429aa3ae8a8e" src="https://user-images.githubusercontent.com/118666111/234497137-2ce4c709-0602-4f7b-b554-e1e60b3ab7c1.png">

**Comment by njroussel** [comment, marked as answer]
Hi @helloKOGOU 

So I think this is a typo: the database will give you a ".pbsdf" file which you can just pass to Mitsuba without any changes. The ".pbrdf" extension is meaningless, the binary content is what matters.
The authors' website also provides a script to load the files in Pyhton and a README, these should give you an idea of how the file layout is.

**Reply by 1405748640-art** [reply]
Hello! I have also been using .pbsdf files to create datasets recently, and the original paper mentions the separation of specular reflection and diffuse reflection. I would like to ask how this part can be implemented in Mitsuba 3?
